ActindoWorkFlow/Operation/03_CompareWorkflows.md

```diff
@@ -0,0 +1,178 @@
+# Compare the workflows versions
+
+The comparison function of the workflow enables to compare different workflow versions. By doing this, you can compare the JSON code lines of two different workflow versions, whereby all differences between the relevant code lines are highlighted.  
+
+You can use this function to do the following: 
+- Compare a specific workflow version to any other version of that workflow available on the current instance.
+- Compare a specific workflow version to any workflow version available on another instance. 
+- Change, insert, or delete code lines. This is possible, because you can edit the JSON code directly.  
+- Display change logs. By doing it, you are able to do software-audits.
+
+## Open the comparison window
+
+Open the comparison window to compare different workflow versions. There are two different ways to open it.
+
+#### Prerequisites
+
+You have the required rights to edit a workflow.
+
+#### Procedure
+
+*Process Orchestration > Workflows > Tab OVERVIEW*
+
+![Workflows](../../Assets/Screenshots/ActindoWorkFlow/Workflows/Workflows.png "[Workflows]")
+
+Select the workflow for which you want to do a comparison.   
+The *Workflow versions* view is displayed.
+    
+![Workflow versions](../../Assets/Screenshots/ActindoWorkFlow/Workflows/WorkflowVersions.png "[Workflow versions]")
+
+**Open the comparison window via the Workflow versions view**
+
+1. Click the checkbox at the desired version you want to use for comparison.   
+    The editing toolbar is displayed.
+    
+2. Click the [COMPARE] button.  
+    The comparison window is displayed. 
+
+    ![Workflow](../../Assets/Screenshots/ActindoWorkFlow/Workflows/WorkflowComparison.png "[Workflow]")  
+
+**Open the comparison window in the workflow editor**   
+
+1. Select the workflow version.   
+    The workflow editor is displayed.
+
+2. Click the ![Points](../../Assets/Icons/Points02.png "[Points]") (Points) button in the upper left corner next to the workflow name.  
+    The workflow context menu is displayed.
+
+    ![Context menu](../../Assets/Screenshots/ActindoWorkFlow/Workflows/ContextMenu.png "[Context menu]")
+
+3. Select the *Show changes* menu entry.  
+    The comparison window is displayed. 
+
+    ![Workflow](../../Assets/Screenshots/ActindoWorkFlow/Workflows/WorkflowComparison.png "[Workflow]")
+    
+
+## Compare the versions
+
+Compare a workflow version with any other version available on the current instance.
```

Comment:
Wenn wir uns weiter unten aus *Compare versions of different instances* auf diese Prozedur beziehen, stimmt der erste Satz dann nicht, obwohl alle hier beschriebenen Schritte passen in beiden Fällen (also, Compare workflow versions in an instance und Compare workflow versions of different instances).       
Vorschlag: Compare a workflow version with any other version available on the current or other instance... (o.ä.)